digits/model/images/classification/test_views.py

```diff
@@ -825,6 +854,17 @@ class BaseTestCreatedTall(BaseTestCreated):
 class BaseTestCreatedCropInForm(BaseTestCreated):
     CROP_SIZE = 8
 
+class BaseTestCreatedDataAug(BaseTestCreated):
```

Comment:
if you inherit from `BaseTestCreatedTall` you'll get to test this on non-square images. Wouldn't that expose the issue on line 54 of data.lua?